openwisp_radius/migrations/__init__.py

```diff
@@ -18,6 +18,10 @@ def add_default_organization(apps, schema_editor):
         for record in Model.objects.all().iterator():
             record.organization_id = settings._OPENWISP_DEFAULT_ORG_UUID
             record.save()
+    Organization = apps.get_model('openwisp_users', 'Organization')
+    OrganizationRadiusSettings = apps.get_model('openwisp_radius', 'organizationradiussettings')
+    org = Organization.objects.get(id=settings._OPENWISP_DEFAULT_ORG_UUID)
```

Comment:
Instead of using `UUID` can you please generate and use a random token, I just think exposing `UUID` to freeradius may come back to bite us. What do you think? :smile:

Reply:
I think you got something wrong there, this line is used to get the default organisation by the uuid. A random token is generated as a token is not provided to the  `OrganizationRadiusSettings` object below this line and it by default generates a random token.

Reply:
Yes, you are correct. Sorry about that.